qa/automation/CareerPortal.Tests.Scenarios/Career Portal/Send your CV.feature

```diff
@@ -0,0 +1,170 @@
+
+Scenario: 1.3 User can select the "Message" text field
+	When User clicks the "Message" text field 
+	Then the "Message" text field is available for text entry
+
+Scenario: 1.4 User can fill the "I agree to the personal data processing" checkbox
+	When User clicks the "I agree to the personal data processing" checkbox
+	Then the "I agree to the personal data processing" checkbox is checked
+
+Scenario: 1.5 User can attach a CV file 
+	When User clicks the "Attach CV" button
+	Then modal window with available device files opens
+
+Scenario Outline: 2.1.1 "Name" text field is valid
+	When "Name" text field filled in with <text>
+	Then line under "Name" text field should turn green
+
+	Examples:
+    | text |
+    |"Name Name"|
+	|"��� ���"| 
+
+Scenario Outline: 2.1.2 "Name" text field is not valid
+	When "Name" text field filled in with <text>
+	Then red "Invalid name format" message box should appear under the "Name" text field
+	
+	Examples:
+    | text |
+    |"    "|  
+    | "11" |
+	| "a"  |
+	|"/Name"|
+	|"Name ���"|
+
+Scenario: 2.2.1 "Name" text field with 71 symbols
+	When "Name" field filled in with "Name Namenamenamenamenamenamenamenamenamenamenamenamenamenamenamenamena"
+	Then red "The maximum number of characters is 70" message box should appear under the "Name" text field
+
+Scenario: 2.2.2 "Name" text field with 70 symbols
+	When "Name" field filled in with "Name Namenamenamenamenamenamenamenamenamenamenamenamenamenamenamenamen"	
+	Then line under "Name" text field should turn green
+
+Scenario: 2.2.3 "Name" text field with 69 symbols
+	When "Name" field filled in with "Name Namenamenamenamenamenamenamenamenamenamenamenamenamenamenamename"
+	Then line under "Name" text field should turn green	
+
+Scenario Outline: 3.1.1 "Email" text field is valid
+	When "Email" text field filled in with <text>
+	Then line under "Email" text field should turn green
+
+	Examples:
+	|     text    |
+	| test@test.ru|
+	|test@test.com|
+
+Scenario Outline: 3.1.2 "Email" text field is not valid
+	When "Email" text field filled in with <text>
+	Then red "Invalid email format" message box should appear under the "Email" text field
+	
+	Examples:
+	|     text    |
+	| test@@test.com|
+	| test)@test.ru |
+	| test*@test.com|
+	| test@����.com |
+	|test..@test.com|
+	| .test@test.ru |
+
+Scenario: 3.2 �onfirmation when the email is correct
+	Given User has entered the correct email
+	And User fills in all of the required fields
+	When User clicks the "Send" button
+	Then a letter of confirmation should be send to the specified email
+ 
+Scenario Outline: 4.1 User can open a valid CV file 
+	When User clicks the "Attach CV" button
+	Then modal window with available <format> device files opens
+
+	Examples:
+	|format|
+    | PDF  |
+	| DOC  | 
+	| DOCX |
+	| RTF  | 
+
+Scenario Oitline: 4.2 User can attach only a valid CV file
+	When User attaches a <format> <size> file
+	Then file attachment should be <status>
+	
```

Comment:
I got what you meant but not sure about that.
Now i'm thinking that "attach cv" button won't let user choose invalid file format (there won't be any in a modal window). Perhaps it would be better to reformulate "When" on "User tries to attach..."?